public/app/features/dashboard/partials/settings.html

```diff
@@ -121,6 +121,17 @@
         <div class="tight-form">
           <ul class="tight-form-list">
             <li class="tight-form-item" style="width: 120px">
+              Created by:
+            </li>
+            <li class="tight-form-item" style="width: 180px">
+              {{dashboardMeta.authorEmail}}
```

Comment:
As advised previously to Claire, to me we must display Firstname / Last name if it's available.
Please manage the metadata AuthorDisplayName instead of authorEmail with the following rule:
AuthorDisplayName = <Author First name> <Author LastName> if this information is available or = authorEmail